recipe/build-pkg.sh

```diff
@@ -7,15 +7,19 @@ else
     FAISS_ENABLE_GPU="OFF"
 fi
 
+if [ ${osx} ]; then
```

Comment:
Ah, didn't notice this before; this won't work -- but we should be able to come up with a solution that works on both Linux and macOS as well, anyway ;).

Reply:
Yeah, I'm in the process of fixing that

Reply:
Not sure why linux works when osx doesn't. But if cmake behaves differently between the two, I have zero problems to have a switch in the build script that makes up for the difference.